frontend/components/MainPageV1/Sidebar.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  PlusOutlined,
+  QuestionCircleOutlined,
+  SettingOutlined,
+} from '@ant-design/icons';
+import { Button, Flex, Layout, Menu, MenuProps, Spin, Typography } from 'antd';
+import Image from 'next/image';
+import { useMemo } from 'react';
+import styled from 'styled-components';
+
+import { ACTIVE_AGENTS } from '@/config/agents';
+import { TOP_BAR_HEIGHT } from '@/constants/width';
+import { Pages } from '@/enums/Pages';
+import { usePageState } from '@/hooks/usePageState';
+import { useServices } from '@/hooks/useServices';
+
+const { Sider } = Layout;
+const { Text } = Typography;
+
+const SiderContainer = styled.div`
+  display: flex;
+
+  .ant-layout-sider {
+    margin-top: -${TOP_BAR_HEIGHT}px;
+    margin-bottom: -${TOP_BAR_HEIGHT}px;
```

Comment:
@mohandast52 this was the simplest way to have the navbar with draggable line on top in order to be able to move the window. you will probably need to use the same approach for some of the pages for onboarding, unless you have better ideas 🙏